electron-test/wdio.conf.ts

```diff
@@ -116,7 +119,12 @@ export const config: Options.Testrunner = {
     // Services take over a specific job you don't want to take care of. They enhance
     // your test setup with almost no effort. Unlike plugins, they don't add new
     // commands. Instead, they hook themselves up into the test process.
-    services: ['electron'],
+    services: [
+        'electron',
+        [ServerLauncher, {
+            somsomeOption: true
```

Comment:
I think this is a typo? 
```suggestion
            someOption: true
```